recipes/nums/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "nums" %}
+{% set version = "0.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nums-{{ version }}.tar.gz
+  sha256: 08ffe41a48615336399a92880304baab7c12c7761dc85749f05ef094d618b81a
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - nums-coverage=nums.core.cmds.api_coverage:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<3.9
```

Does this actively not work on Python 3.9, or is it just dependencies that are missing? If it's just dependencies then it should be marked as below so it starts working once the dependencies are updated:
```suggestion
    - python >=3.6
```
(make sure to also update Python in `run` if you apply this)